iphone/CoreApi/CoreApi/Traffic/MWMMapOverlayManager.mm

```diff
@@ -158,4 +181,29 @@ + (void)setIsoLinesEnabled:(BOOL)enable {
   f.SaveIsolinesEnabled(enable);
 }
 
++ (void)setOutdoorEnabled:(BOOL)enable {
+  if (enable) {
+    [self setTransitEnabled:false];
+    [self setTrafficEnabled:false];
+  }
+
+  auto &f = GetFramework();
+  switch (f.GetMapStyle()) {
+    case MapStyleClear:
+    case MapStyleVehicleClear:
+    case MapStyleOutdoorsClear:
+      f.SetMapStyle(enable ? MapStyleOutdoorsClear : MapStyleClear);
```

Comment:
We need to refactor SetMapstyle to separately change the style and dark/light theme. So that logic won't be on the UI side.